apps/docs/content/components/Hero.mdx

```diff
@@ -18,53 +27,73 @@ import {Hero} from '@primer/react-brand'
-| `description`                           | `string`                                                                                                     |           | Optional accompanying body text          |
-| `className`                             | `string`                                                                                                     |           | Sets a custom class on the root element  |
-| `id`                                    | `string`                                                                                                     |           | Sets a custom id                         |
-| `ref`                                   | `React.RefObject`                                                                                            |           | Forward a Ref to the underlying DOM node |
+| Name        | Type                  |  Default  | Description                              |
+| :---------- | :-------------------- | :-------: | :--------------------------------------- |
+| `align`     | `'center'`, `'start'` | `'start'` | Optional text alignment                  |
+| `className` | `string`              |           | Sets a custom class on the root element  |
+| `ref`       | `React.RefObject`     |           | Forward a Ref to the underlying DOM node |
+
+### Hero.Heading
+
+| name            | type                                                                                               | default | description                                    |
+| --------------- | -------------------------------------------------------------------------------------------------- | ------- | ---------------------------------------------- |
+| `className`     | `string`                                                                                           |         | Sets a custom class on the root element        |
+| `id`            | `string`                                                                                           |         | Sets a custom id                               |
+| `ref`           | `React.RefObject`                                                                                  |         | Forward a Ref to the underlying DOM node       |
+| `size`          | <PropTableValues values={HeadingSizes} addLineBreaks />                                            |         | Override the default visual text size          |
+| `weight`        | <PropTableValues values={[...HeadingWeights, 'ResponsiveWeightMap']} addLineBreaks />              |         | Override the default visual text weight        |
+| `stretch`       | <PropTableValues values={[...HeadingStretch, 'ResponsiveStretchMap']} addLineBreaks />             |         | Override the default visual text stretch value |
+| `letterSpacing` | <PropTableValues values={[...HeadingLetterSpacing, 'ResponsiveLetterSpacingMap']} addLineBreaks /> |         | Override the default visual character spacing  |
```

Comment:
For maintainability reasons, I like what we do in `SectionIntro` by adding a description and pointing to the `Heading` documentation. 

https://primer.style/brand/components/SectionIntro#sectionintroheading-required

Reply:
+1 to this☝️🙏

Reply:
Great suggestion! Did this for `Hero.Description` too.